### PR TITLE
Release SDK 1.2.0

### PR DESCRIPTION
**Pull request by cocreature**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/support/release-notes.rst [hunk 1]
```diff
@@ -6,9 +6,158 @@ Release notes
...
+   translating to and from the serialized DAML-LF values when fetched
+   from the participant index.
+-  Added new Ledger API Server metrics
+   for ``daml.index.db.*.deserialization`` to measure the duration of the
+   translation of the serialized DAML-LF values when fetched from the
+   participant index.
+-  The Ledger API Test Tool has gained
+   the ``TransactionSize`` performance benchmark test.
+
+What’s Coming
```
Comment (bame-da) [inline docs/source/support/release-notes.rst:139]: I don't think this should go into our docs, but just the blog.

Comment (cocreature) [reply docs/source/support/release-notes.rst:139]: We always included it in the past. Happy to stop doing it from now on if you want.

Comment (cocreature) [reply docs/source/support/release-notes.rst]: removed

**Comment by cocreature** [comment]
Manual tests passed on Linux and Windows.

**Pull request merged**